aten/src/ATen/function_wrapper.py

```diff
@@ -427,6 +427,7 @@ def __getitem__(self, x):
 
 ReturnType = TypedDict('ReturnType', {
     'name': str,
+    'field_name': str,
```

Comment:
What's the difference between `name` and `field_name`?

Reply:
@ezyang  Return values of ATen operators always has a `name`. If not explicitly assigned a name inside `native_functions.yaml`, then the codegen will automatically assign it a name like `result0`, or it might be specified inside `Declarations.cwrap`. We don't want these names assigned names to become part of API. So I created `field_name`, which is defined only when there is a name specified in `native_functions.yaml`. If `field_name` is defined, then the codegen would generate code to return namedtuple. Otherwise, it would just return tuple.